tests/form.js

```diff
@@ -527,7 +527,61 @@ define([
                     }
                 });
             });
+        });
+
+        describe("hashtag logic", function () {
+            var form;
+            before(function () {
+                form = util.loadXML("");
+            });
+
+            _.each({
+                "": null,
+                "#": null,
+
+                "#form": null,
+                /* TODO should these pass? (they don't)
+                "#form/": "is",
+                "#form/prop": "has",
```

Comment:
From my memory. I think  this may be due to only checking valid hashtags against #case due to being interweaved with the data sources code, agreed with your  comment that they should pass